features/step_definitions/us109_conveyancer_confirms_identity_checks.rb

```diff
@@ -0,0 +1,17 @@
+Given(/^conveyancer is not confirming borrower identity check$/) do
+  @deed = Deed.new(1)
+  @deed.identity_checked = 'N'
+  @deed_hash = @deed.to_hash
+end
+
+Given(/^conveyancer has submitted an invalid identity check value$/) do
+  @deed = Deed.new(1)
+  @deed.identity_checked = 'Blah'
```

Comment:
Test has been removed now as it was doing the same as the test with a value of 'N'